lib/PostMessageTransport.js

```diff
@@ -135,10 +135,13 @@ define(function (require, exports, module) {
 
     /**
      * Returns the script that should be injected into the browser to handle the other end of the transport.
+     * Includes a base tag to handle external protocol-less, linked files.
      * @return {string}
      */
     function getRemoteScript() {
-        return "<script>\n" +
+        var base = window.location.href;
```

Comment:
Get rid of the `base` variable and just do it inline like this: `return '<base href="' + window.location.href + '">\n' +`.  Also note that I've been careful to wrap it in double-quotes.